test/e2e/certificates_test.go

```diff
@@ -354,6 +355,50 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("OCP-65035-CLOUD_CREDENTIALS_SECRET_NAME works well in mint/passthrough clusters", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
+
+			By("getting Infrastructure object")
+			infra, err := configClient.Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			if infra.Status.PlatformStatus.Type != configv1.GCPPlatformType {
+				Skip("Skip this case for current plaform, it's not gcp.")
+			}
+
+			By("Check cloud credential in cluster")
+			_, err = loader.KubeClient.CoreV1().Secrets("cert-manager").Get(ctx, "gcp-credentials", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
we should do this after `By("creating CredentialsRequest object")`, and need apply a `wait-poll` like: https://github.com/openshift/cert-manager-operator/blob/46069515b0bda3a5f51ba88705095a089258d07b/test/e2e/certificates_test.go#L185-L193

Reply:
@lunarwhite Did you confuse the credential and credentialrequest? get gcp-credential is first step in test case [ocp-65035](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitems/testcase?query=id%3AOCP%5C-65035), the 2nd step is create CredentialsRequest

Reply:
@geliu2016, @lunarwhite is right. Need to update as per @lunarwhite 's `wait` example. Above `wait` example is from Dev's existing code for the AWS case. Just click above link " [cert-manager-operator/test/e2e/certificates_test.go Lines 185 to 193 in 4606951](https://github.com/openshift/cert-manager-operator/blob/46069515b0bda3a5f51ba88705095a089258d07b/test/e2e/certificates_test.go#L185-L193) ".

Reply:
@xingxingxia regarding link " [cert-manager-operator/test/e2e/certificates_test.go Lines 185 to 193 in 4606951](https://github.com/openshift/cert-manager-operator/blob/46069515b0bda3a5f51ba88705095a089258d07b/test/e2e/certificates_test.go#L185-L193) ", you may find there is not verification after:
By("creating CredentialsRequest object")
loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "credentials", "credentialsrequest_aws.yaml"), "")

and the wait.PollImmediate is check "Secrets("cert-manager").Get(ctx, "aws-creds", metav1.GetOptions{})", we also have this secret check at beginning:

By("Check cloud credential in cluster")
_, err = loader.KubeClient.CoreV1().Secrets("cert-manager").Get(ctx, "gcp-credentials", metav1.GetOptions{})

Reply:
@geliu2016 , no, they are different things.
The beginning checks a `system` secret within `kube-system` namespace. But here we need check a secret defined in your credentialsrequest to be generated within the `cert-manager` namespace. Please check carefully the link again, it shows `("cert-manager")` namespace:
```
_, err := loader.KubeClient.CoreV1().Secrets("cert-manager").Get(ctx, "aws-creds", metav1.GetOptions{})
```

Reply:
updated.